.env.vagrant

```diff
@@ -32,5 +32,7 @@ OAUTH2_TOKEN_URI=http://web.chs-dev.internal:4000/oauth2/token
 OAUTH2_CLIENT_ID=
 OAUTH2_CLIENT_SECRET=
 
+ALLOWED_COMPANY_PREFIXES=SC,NI,OC,SO,R,AP
```

Comment:
Do we have a PR for updating all the default enviroment configs in the chs-config repo?

Reply:
https://github.com/companieshouse/chs-configs/pull/1074/files

Reply:
I'm wondering whether it would be a good idea to add a note to the config ensuring 'AP' does not make it to the live environment?

Reply:
could even have a separate property if we are worried this will make it into live

Reply:
Surely on the release notes we just have to list what should be on live.

Reply:
yep, that's fine as long as someone remembers this